docs/examples.rst

```diff
@@ -3,9 +3,6 @@ Examples
 
 In the following is a list of examples you can study and copy.
 
-First is a snippet of the ``global_variables.yml`` file which holds the static
-metadata. In real cases this file will be much longer.
-
 Then there are several examples for various datatypes. In general there is a python
 script, and next is an example from a metadata file that is produced by
 the script.
```

Comment:
Rephrase slightly? E.g.
`The examples covers various datatypes. The examples typically shows a python script, together with a corresponding metadata file that would be produced when running the script.`